app/assets/javascripts/spotlight/admin/visibility_toggle.es6

```diff
@@ -0,0 +1,23 @@
+// Visibility toggle for items in an exhibit, based on Blacklight's bookmark toggle
+// See: https://github.com/projectblacklight/blacklight/blob/main/app/javascript/blacklight/bookmark_toggle.js
+
+const CheckboxSubmit = spotlightAdminCheckbox_submit
```

Comment:
Can you explain where this comes from?

Reply:
Yeah, this is pretty weird...we tell sprockets to re-export things as UMD to the global scope. It's from https://github.com/projectblacklight/spotlight/pull/2599. I'll add a comment about it

Reply:
Added